jablib/src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -73,32 +73,38 @@ public String layout(String layoutFile, BibEntry entry) throws IOException {
         return layout.doLayout(entry, null);
     }
 
-    @Test
-    void parseMethodCalls() {
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla,").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("_bla.bla.blub,").size());
-        assertEquals("_bla.bla.blub", LayoutEntry.parseMethodsCalls("_bla.bla.blub,").getFirst().getFirst());
-
-        assertEquals(2, LayoutEntry.parseMethodsCalls("bla,foo").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,foo").getFirst().getFirst());
-        assertEquals("foo", LayoutEntry.parseMethodsCalls("bla,foo").get(1).getFirst());
+    @ParameterizedTest
+    @CsvSource(textBlock = """
+            bla, bla
+            bla, bla
```

Comment:
I didn't see in the first review that the second test case  was `bla,'. Note the final comma! You removed it while converting. I think, it should be possible to get this in again with following change:

```suggestion
            bla, 'bla,'
```